src/qibo/tests/test_qasm.py

```diff
@@ -90,6 +118,20 @@ def test_toffoli():
     assert_strings_equal(c.to_qasm(), target)
 
 
+def test_toffoli_cirq(atol=1e-7):

```

Comment:
When the same `atol` is required for several tests in a file, I usually define a global `_atol = 1e-7` variable and use `atol=_atol` in each function. This is in order to try different `atol` values without having to change each test separately.

Reply:
Sure, good point, thanks.